src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useContext } from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+import { DispatchContext, StateContext } from '../../context/ReduxContext';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { currentId } = useContext(StateContext);
+  const dispatch = useContext(DispatchContext);
+
+  const { id, title, completed, userId } = todo;
+
+  const handleSeeClick = () => {
+    dispatch({
+      type: 'seeMore',
+      curruntId: id,
+      currentTitile: title,
+      statusComleted: completed,
+      userId: userId,
+    });
```

Comment:
```suggestion
    dispatch({
      type: 'seeMore',
      payload: todo
    });
```

1. move all your fields to the payload field
2. just save the selectedTodo